### ISSUE TITLE
Feature request: support Youtube RSS feed

### ISSUE THREAD

**Issue by xxyzz** [brentsimmons/NetNewsWire#1167]
YouTube channels can be added to the app, but when I click them there just a title and the date. Can NetNewsWire load the video inside the app so I don't have to click the title and watch it on my browser?

The feed URL of the screenshot is https://www.youtube.com/feeds/videos.xml?channel_id=UC2C_jShtL725hvbm1arSV9w 

<img width="1190" alt="Screen Shot 2019-10-23 at 22 29 58" src="https://user-images.githubusercontent.com/21101839/67403530-a7ea2780-f5e4-11e9-8d2f-0cdec30019a1.png">

**Comment by vincode-io** [comment]
We are compatible with Youtube if NetNewsWire gets the correct feed.

It looks like we need to tweak the NNW local accounts Feed Finder to work with Youtube better.  We end up finding the feed that @xxyzz is looking for if we use the Feedbin add account functionality.

![Screen Shot 2019-10-23 at 9 48 07 AM](https://user-images.githubusercontent.com/16448027/67405348-3d9c9d00-f57a-11e9-8fdb-029df502173f.png)

**Comment by zucktom** [comment]
Any update on this?

**Comment by teroyks** [comment]
Yes please! It would be nice if I could just paste the channel URL `https://www.youtube.com/channel/CHANNEL_ID` in NetNewsWire instead of having to manually change it to the feed URL `https://www.youtube.com/feeds/videos.xml?channel_id=CHANNEL_ID`.

**Comment by sorenpeter** [comment]
And even manually changing the URL as Teroyks says it not solving the issue of not showing the video: 
![Screenshot 2020-11-15 at 10 50 34](https://user-images.githubusercontent.com/8458421/99181706-7765b600-2730-11eb-935b-28d67dba0559.png)

Feed used is: https://www.youtube.com/feeds/videos.xml?channel_id=UCAPR27YUyxmgwm3Wc2WSHLw

**Comment by xxyzz** [comment]
Now I prefer to watch the video on IINA(https://github.com/iina/iina) instead of a browser.

**Comment by teroyks** [comment]
I use IINA to watch the videos as well but I use NetNewsWire to keep track of new videos. 

It would be great if I could configure a keyboard shortcut to copy the article link URL to clipboard (instead of having to Ctrl-click on the title) btw!

**Comment by nsklaus** [comment]
being able to define which player to send the link to, would be even better.
like we can choose which browser we want to use, if we could also choose which player it would be nice.
there is iina, or plain mpv, or vlc too.. personally i use plain mpv.
reeder allows this. in the list of possible browsers it proposes mpv too.

**Comment by stn-patrick** [comment]
i'm using [choosy](https://www.choosyosx.com) for this, but a native feature would be great. thank you for the great app

**Comment by Krinkle** [comment]
RE: NNW failing to find the feed from a channel page, this was also reported at https://github.com/Ranchero-Software/NetNewsWire/issues/902, and has been solved.

RE: Feed content blank/empty after you follow a YouTube channel, that is how YouTube provides the feed. I guess they like making it difficult, or maybe the RSS feature dates back to the Flash days and hasn't been updated since. There is a proposal for NNW to locally enhance these feeds and generate an inline player, tracked at: https://github.com/Ranchero-Software/NetNewsWire/issues/3683. 

RE: Opening YouTube.com URLs in another app, I assume that's the subject of this issue and remains open.
